src/common/LogClient.cc

```diff
@@ -126,7 +130,7 @@ LogChannel::LogChannel(CephContext *cct, LogClient *lc,
 {
 }
 
-LogClient::LogClient(CephContext *cct, Messenger *m, MonMap *mm,
+LogClient::LogClient(CephContext *cct, messenger_t *m, MonMap *mm,
```

Comment:
i'd avoid acquiring POSIX locks in crimson. if it's fine by you, i'd like to reimplement LogClient just for crimson.